src/main/java/ch/wisv/chue/states/HueState.java

```diff
@@ -0,0 +1,13 @@
+package ch.wisv.chue.states;
+
+import ch.wisv.chue.HueCommand;
+
+/**
+ * @author Sander Ploegsma
+ */
+public abstract class HueState extends HueCommand {
+
+    public HueState() {
+
+    }
```

Comment:
Mja ik wist niet zeker of er nog onderscheid gemaakt moest worden tussen States en Events, dus vandaar dat ik die twee nog heb gelaten. Qua functionaliteit kan je nu alleen zeker weten dat States als states worden geladen en events als events :stuck_out_tongue_winking_eye: